fastkml/validate.py

```
@@ -0,0 +1,101 @@
+                    0
+                ].getparent()
+            except config.etree.XPathEvalError:
+                parent = element
+            error_in_xml = config.etree.tostring(
+                parent,
+                encoding="UTF-8",
+                pretty_print=True,
+            ).decode(
+                "UTF-8",
+            )
+            logger.error("Error <%s> in XML:\n %s", e.message, error_in_xml)
```

Ruff

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)